docs/how_to/support.rst

```diff
@@ -6,39 +6,30 @@ Community
    :is_head: true
    :link: https://www.apache.org/foundation/policies/conduct
 
-   Welcome to GluonNLP community. We strive to foster a collaborative and welcoming community. We
+   Welcome to GluonCV community. We strive to foster a collaborative and welcoming community. We
    expect all members to follow the `code of conduct <https://www.apache.org/foundation/policies/conduct>`__.
 
 
 .. container:: cards
 
    .. card::
       :title: Github Issues
-      :link: https://github.com/dmlc/gluon-nlp/issues
+      :link: https://github.com/dmlc/gluon-cv/issues
 
       Feature requests, bug reports, design and roadmap discussion.
 
 
    .. card::
       :title: Github Projects
-      :link: https://github.com/dmlc/gluon-nlp/projects
+      :link: https://github.com/dmlc/gluon-cv/projects
 
       Active roadmaps, and current and past projects.
 
-
-   .. card::
-      :title: GluonNLP Slack Channel
-      :link: https://apache-mxnet.slack.com/messages/CCCDM10V9
-
-      #gluon-nlp Slack channel. Click the `sign-up link <https://join.slack.com/t/apache-mxnet/shared_invite/enQtNDQyMjAxMjQzMTI3LTkzMzY3ZmRlNzNjNGQxODg0N2Y5NmExMjEwOTZlYmIwYTU2ZTY4ZjNlMmEzOWY5MGQ5N2QxYjhlZTFhZTVmYTc>`_ to register.
```

Comment:
It's easy to set one up. Would you like that?

Reply:
Do you have the permission?

Reply:
Yes, this is on the old site.

Reply:
Here you go: https://apache-mxnet.slack.com/messages/CED6Y10E5